cmd/scsd/creds.go

```diff
@@ -1112,3 +1130,49 @@ func doCredsGet(w http.ResponseWriter, r *http.Request) {
 	w.Write(ba)
 }
 
+func doHSMDiscover(xnames []string) ([]byte, error) {
+	if len(xnames) == 0 {
+		return nil, nil
+	}
+	logger.Error("********** IN HSM DISCOVER ****************")
+	if hsmClient == nil {
+		hsmTransport = &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}
+		hsmClient = &http.Client{Transport: hsmTransport}
+	}
+	url := appParams.SmdURL + "/Inventory/Discover"
+	logger.Error(url)
+
+	var payloadS discoverPayload
+	payloadS.Xnames = xnames
+	payloadS.Force = true
+	//	payload := "{\"xnames\":[\"" + xname + "\"], \"force\":true}"
```

Comment:
Can we remove this comment, I don't think we need it?

Reply:
Updated